Taruna

TarunaC opened the pull request:
Removed the Assignments project and Added the login functionality of Docusign project

TarunaC committed:
Merge pull request #1 from primelearning/master

Merge pull request #1 from TarunaC/Taruna

TarunaC committed:
added Loginpage basetest and login page class in framework

TarunaC committed:
removed Assignments folder from framework

avighub commented on src/main/java/PageObjects/LoginPage.java:
```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage {
```
make it public final class so nobody can extend it.

avighub commented on src/main/java/PageObjects/LoginPage.java:
```diff
@@ -0,0 +1,34 @@
+
+
+    public void clickLoginLink(){
+        localDriver.findElement(By.xpath("//a[text()='Log In']")).click();
```
You did not use the loginLink variable. replace hardcoding locator with variable.

avighub commented on src/test/java/FunctionalTest/LoginPageTest.java:
```diff
@@ -0,0 +1,29 @@
+    login.clickOnNext();
+    login.enterPassword("TestPractice@2");
+    login.clickOnNext();
+   // String loginVerification=driver.findElement(By.xpath("//button[@data-qa='header-HOME-tab-button']")).getText();
```
never leave unused code, if you have unfinished code below approach is advised
//TODO: Some work is pending here, so you can come back and finish

avighub reviewed (commented):
Please check comments and fix them and raise PR again

TarunaC committed:
Changes made as per PR comments.
- remove commented code from LoginPageTest class
-LoginPage class is made final and Hardcoded variable is removed.

TarunaC committed:
Added Properties files and Classes to read and load the data from the files.
made changes in LoginPage class accordingly.
